crmd/lrm.c

```diff
@@ -2077,6 +2077,19 @@ do_lrm_rsc_op(lrm_state_t * lrm_state, lrmd_rsc_info_t * rsc, const char *operat
         }
     }
 
+    if (!is_remote_lrmd_ra(NULL, NULL, rsc->id)) {
```

Comment:
Of course, I wasn't thinking it through -- it does supply metadata with a version, but lrmd doesn't execute it :-)